README.md

```diff
@@ -188,6 +212,8 @@ Product.by(id: '1', name: 'Demo Product')
 
 ```
 
+:exclamation: There is no `add_raw_mutation` since `add_raw_query` does already cover such a case
```

Comment:
Does it make sense to only create an alias?

Reply:
> Does it make sense to only create an alias?

Do you mean that we have `add_raw_mutation` which is basically `add_raw_query`? If so then I would say we should avoid this. Basically it could lead to the following:

```ruby
add_raw_mutation 'query { products { id name } }'
```

Whereas this feels kind of "ok":

```ruby
add_raw_query `mutation { createProducts(id: "Foo", name: "Bar") { id name } }
```

Another option would be to have smth like `add_raw_...` which covers both name. 

E.g. `add_raw_graphql`